p2p/libp2p/discovery/mdnsDiscoverer_test.go

```diff
@@ -114,7 +114,7 @@ func TestMdnsPeerDiscoverer_ApplyContextShouldWork(t *testing.T) {
 	//TODO delete skip when mdns library is concurrent safe
 	t.Skip("mdns library is not concurrent safe (yet)")
 	ctx, _ := libp2p2.NewLibp2pContext(context.Background(), &mock.ConnectableHostStub{})
-	mdns := discovery.NewMdnsPeerDiscoverer(time.Duration(time.Second*1), "srvcTag")
+	mdns := discovery.NewMdnsPeerDiscoverer(time.Second*1, "srvcTag")
```

Comment:
time.second is enough "*1" is in plus

Reply:
agree

Reply:
Searched and removed from entire project